pom.xml

```diff
@@ -122,6 +122,26 @@
         <version>4.4.13</version>
         <scope>test</scope>
       </dependency>
+
+      <!-- Test dependencies -->
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>4.13</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.google.truth</groupId>
+        <artifactId>truth</artifactId>
+        <version>1.0.1</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-iam-v1</artifactId>
+        <version>0.13.0</version>
```

Comment:
This one also seems like it should be managed by the shared-deps

Reply:
ok - adding this to shared-deps https://github.com/googleapis/java-shared-dependencies/pull/55

Reply:
done